### PR TITLE
test #2543

### PR DESCRIPTION
**Pull request by briceruzand**
I only provide the test case to reproduce the bug #2543 

#### Checklist

- [ ] fix done
- [ ] run `npm run test` and `npm run benchmark`
- [X] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/content-length.test.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ test('default 413 with bodyLimit option', t => {
     bodyLimit: 10
   })
 
-  fastify.post('/', function (req, reply) {
+  fastify.post('/', function (_req, reply) {
```
Comment (jsumners) [inline test/content-length.test.js:14]: Why have all of these parameters been renamed?

Comment (fox1t) [reply test/content-length.test.js:14]: I suppose that it is because they are not used in the function.

Comment (briceruzand) [reply test/content-length.test.js:14]: sorry eslinter do it for me

Comment (jsumners) [reply test/content-length.test.js:14]: Please use the linter configured for the project and revert these unnecessary changes.

File: test/content-length.test.js [hunk 2]
```diff
@@ -160,3 +160,30 @@ test('#2214 - wrong content-length', t => {
...
+    throw error
+  })
+
+  fastify.inject({
+    method: 'GET',
+    path: '/'
+  })
+    .then(res => {
+      t.strictEqual(res.statusCode, 500)
+      t.strictEqual(res.headers['content-length'], '' + res.rawPayload.length)
```
Comment (Eomm) [inline test/content-length.test.js:185]:
```suggestion
      t.same(res.headers['content-length'], res.rawPayload.length)
```

**Comment by RafaelGSS** [comment]
I created a PR for it #2546

**Comment by briceruzand** [comment]
Thx @RafaelGSS your PR #2546 seems to be good.

Duplicate of #2546

**Pull request closed without merging**